flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -525,6 +528,16 @@ private boolean callInsertInto(SqlCommandCall cmdCall) {
 		return true;
 	}
 
+	private void callCreateTable(SqlCommandCall cmdCall) {
+		try {
+			executor.createTable(sessionId, cmdCall.operands[0]);
+		} catch (SqlExecutionException e) {
+			printExecutionException(e);
+			return;
```

Comment:
We may need to invoke `executor.dropTable` for roll back once the `drop table` command is supported. Or do you plan to support drop table in this PR ?